.github/workflows/update-release-notes.yml

```diff
@@ -12,8 +12,26 @@ env:
   user_for_confluence: honeycomb@hitachivantara.com              # user for API token of Confluence
   
 jobs:
+  update_stable_tag:
+    runs-on: [k8s]
+    permissions:
+      contents: write
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          ref: master
+          token: ${{ secrets.HV_ADMIN_GITHUB_TOKEN}} 
+      
+      - run: sudo chmod u+rwx tag.sh 
```

Comment:
Do you need `sudo`? `chmod +x tag.sh` is not enough?

Reply:
No, `chmod +x tag.sh` works fine